setup.py

```diff
@@ -27,10 +27,5 @@
         'console_scripts': [
           'evalml = evalml.__main__:cli'
         ]
-    },
-    data_files=[('evalml/tests/data', ['evalml/tests/data/tips.csv',
-                                       'evalml/tests/data/titanic.csv',
-                                       'evalml/tests/data/churn.csv',
-                                       'evalml/tests/data/fraud_transactions.csv.gz']),
-                ('evalml/demos/data', ['evalml/demos/data/daily-min-temperatures.csv'])],
```

Comment:
I think we should undo this change and add daily-min-temperatures to the list of `evalml/tests/data`. We need this so the data gets published with the pypi package and conda can run the tests.

Reply:
I think we can remove the copy of daily-min-temperatures in demos/data from version control too.

Reply:
Fixed
https://github.com/alteryx/evalml/pull/3023/commits/0b7293b5ecb2fa9d0d002006bcaeda29b59ad83d